modules/operators/migrations/migration.json

```diff
@@ -1,4 +1,10 @@
 {
   "$schema": "../../../node_modules/@angular-devkit/schematics/collection-schema.json",
-  "schematics": {}
+  "schematics": {
+    "20.0.0-beta_1-tap-response": {
+      "description": "Update tapResponse signature",
+      "version": "20.0.0-beta.1",
+      "factory": "./20_0_0-beta_1-tap-response/index"
+    }
+  }
```

Comment:
```suggestion
  "schematics": {
    "20.0.0-rc_0-tap-response": {
      "description": "Replace deprecated tapResponse signature",
      "version": "20.0.0-rc.0",
      "factory": "./20_0_0-rc_0-tap-response/index"
    }
  }
```

Reply:
Updated the other files as well for usage of previous version.